apps/sms/views/conversation/style/message.css

```diff
@@ -79,10 +82,9 @@
   background: #ff9898;
 }
 
-.message.sending .bubble,
-.message.sending a,
-.message.sending a:active {
-  color: #bfbfbf;
+.message.sending .bubble {
+  opacity: 0.5;
+  color: #6e6e6e;
```

Comment:
Mmm with opacity on entire ```.message.sending .bubble``` we change the color of the message details (e.g. ```sending....``` string). I'd say we need to reduce opacity for ```.message.sending .message-subject``` and ```.message.sending .message-content-body``` only. You can compare how  "sending..." looks like on master and how it looks with the patch.